data/strings/types_strings.txt

```diff
@@ -26996,7 +26999,7 @@
   [type.piste_type.downhill.intermediate]
     ref = type.piste_type.downhill
     en = Intermediate Downhill Ski Run
-    de = Mittelschwere Abfahrt
+    de = Mittelschwere Ski Abfahrt
```

Comment:
Ski-Abfahrt

Reply:
To make it easier to apply the suggestion for @map-per ([see here](https://haacked.com/archive/2019/06/03/suggested-changes)):

```suggestion
    de = Mittelschwere Ski-Abfahrt
```